Sources/SotoCore/HTTP/S3StreamWriter.swift

```diff
@@ -0,0 +1,140 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Soto for AWS open source project
+//
+// Copyright (c) 2017-2020 the Soto project authors
```

Comment:
This probably needs to be updated
```suggestion
// Copyright (c) 2017-2021 the Soto project authors
```